pigpen-cascading/src/main/clojure/pigpen/cascading/core.clj

```diff
@@ -190,7 +196,7 @@
   [{:keys [id projections fields]} :- m/Project
    [{:keys [^Pipe pipe ancestor]}]
    _]
-  (let [context (pr-str `'{:ancestor ~(update-in ancestor [:opts] dissoc :tap)
+  (let [context (pr-str `'{:ancestor ~(clojure.walk/postwalk #(if (map? %) (dissoc % :tap) %) ancestor)
```

Comment:
It looks like the load command got munged with the group's opts. That seems like a bug. Do you know where that's coming from? Could you send me an example that produces this?